src/errorReporting.js

```diff
@@ -96,10 +96,19 @@ function getPendingPromise(state) {
   return state.getIn(['errorReporting', 'pendingPromise']);
 }
 
-export function serverSideError(error) {
-  // nodejs console truncates output by default
-  // https://nodejs.org/api/util.html#util_util_inspect_object_options
-  console.error(util.inspect(error, false, 10, true));
+export function serverSideError(queue) {
+  queue.forEach((raw) => {
```

Comment:
maybe add something like this so that if its a single error or array of errors does not matter
```suggestion
  [].concat(queue).forEach((raw) => {
```

Reply:
Sure will add. The type of queue is always an array but this way the function can used somewhere else.